### PR TITLE
PREF-307 | Preloading

### PR DESCRIPTION
**Pull request by Firtzberg**
We'll discuss this during the meeting

## Change Notes
- Added Preloaders for Doctrine and HTTP containers

## JIRA Issues
- [PREF-307](https://55places.atlassian.net/browse/PREF-307)

### CODE REVIEW

File: http/fab/Prefab5/HTTPBuildableDirectoryMap/ContainerBuilder/Preloader.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    private function preloadContainer(ContainerInterface $container): PreloaderInterface
+    {
+        $application = $container->get(Application::class);
+
+        // Application may be run only once.
+        if (!$this->applicationRun) {
+            $this->applicationRun = true;
+            try {
+                $application->run();
+            } catch (Throwable $throwable) {
```
Comment (Firtzberg) [inline http/fab/Prefab5/HTTPBuildableDirectoryMap/ContainerBuilder/Preloader.php:62]: add comment saying that this is expected

File: docs/BuildableDirectories.md [hunk 2]
```diff
@@ -58,3 +58,46 @@ $primer->primeContainers();
...
+[Preloading](https://www.php.net/manual/en/opcache.preloading.php) has been introduced in PHP 7.4.
+
+Preloading the built HTTP containers significantly reduces the response time. In addition to preloading the HTTP containers also preload Doctrine.
+
+First write the preloading script, an example is shown below.
+The script must not start with `#!/usr/bin/env php`!
+
+```php
+<?php
+declare(strict_types=1);
```
Comment (Firtzberg) [inline docs/BuildableDirectories.md]: Must remove this, add note

File: docs/BuildableDirectories.md [hunk 3]
```diff
@@ -58,3 +58,46 @@ $primer->primeContainers();
...
+
+Container priming uses the environment variables and bakes them into the container, therefore container priming should be triggered by the entrypoint script.
+
+## Preloading
+
+[Preloading](https://www.php.net/manual/en/opcache.preloading.php) has been introduced in PHP 7.4.
+
+Preloading the built HTTP containers significantly reduces the response time. In addition to preloading the HTTP containers also preload Doctrine.
+
+First write the preloading script, an example is shown below.
```
Comment (Firtzberg) [inline docs/BuildableDirectories.md:70]: Add executable in prefab's bin folder

Comment (Firtzberg) [reply docs/BuildableDirectories.md:70]: Not possible, since the classes involved are fabricated

**Review by harshpatel991 — APPROVED** [review]

**Pull request merged**